yarn.lock

```diff
@@ -6194,8 +6194,6 @@ __metadata:
 "axios@npm:^0.25.0":
   version: 0.25.0
   resolution: "axios@npm:0.25.0"
-  dependencies:
-    follow-redirects: ^1.14.7
```

Comment:
Things like this should _not_ be removed. This indicates that `axios@0.25.0` depends on `follow-redirects`.